torch_xla/_internal/pjrt.py

```diff
@@ -101,6 +102,12 @@ def _run_singleprocess(fn: Callable[..., R], *args, **kwargs) -> Dict[int, R]:
 
   return fn(*args, **kwargs)
 
+def get_global_rank(local_rank: int):
```

Comment:
This overlaps confusingly with the `global_ordinal` and the `process_index` in `torch_xla.runtime`, and this usage of the `RANK` env var or the local rank strikes me as a special case. `RANK` is not always set.

AFAICT, this index is used in one place: to determine whether to start the distributed runtime. How about something like `is_distributed_master(local_rank: int)` or `should_start_xla_coordinator(local_rank:int)` so we don't have another another index-getter?

cc @jonb377 for input since you're going to need to use this for checkpointing

Reply:
SG

Reply:
done

Reply:
Yeah, it's gone in the latest iteration.